example/server/api/index.js

```diff
@@ -1,32 +1,22 @@
 const Router = require('express').Router;
-const environmentID = 'uCDQzKWgejrutqSYYsKWen';
-const flagsmith = require('../../../');
-const NodeCache = require('node-cache');
+const { Flagsmith } = require('../../../sdk');
 
-flagsmith.init({
-    environmentID
-    // this is an example of a user-defined cache
-    /*cache: new NodeCache({
-        stdTTL: 5
-    })*/
+const flagsmith = new Flagsmith({
+    environmentKey: 'some-key'
 });
 
 module.exports = () => {
     const api = Router();
 
     api.get('/', async (req, res) => {
-        const font_size = await flagsmith.getValue('font_size');
-        res.json({ font_size });
-    });
-
-    api.get('/flags', async (req, res) => {
-        const flags = await flagsmith.getFlags();
+        const flags = await flagsmith.getEnvironmentFlags();
         res.json(flags);
     });
 
     api.get('/:user', async (req, res) => {
-        const font_size = await flagsmith.getValue('font_size', req.params.user);
-        res.json({ font_size });
+        const flags = await flagsmith.getIdentityFlags(req.params.user);
+        const fontSize = flags.getFeatureValue('font_size');
```

Comment:
Since it's vital for enableLocalEvaluation, I think we should pass triats to the getIdentityFlags call